### PR TITLE
GH-23: Adding Tizen Support

### PR DESCRIPTION
**Pull request by mattleibow**
### Description of Change ###

This is a PR to both see what it means to support Tizen as well as to start a discussion about adding Tizen support.

### Bugs Fixed ###

- Related to issue #23

### API Changes ###

No new APIs except for `DeviceInfo.Platforms.Tizen`. The rest is just the Tizen _implementation_ of the existing APIs.

### Behavioral Changes ###

Adding Tizen support.

### PR Checklist ###

- [ ] Has tests (if omitted, state reason in description)
- [ ] Has samples (if omitted, state reason in description)
- [ ] Rebased on top of master at time of PR
- [ ] Changes adhere to coding standard
- [ ] Updated documentation ([see walkthrough](https://github.com/xamarin/Essentials/wiki/Documenting-your-code-with-mdoc))

### API Checklist ###

 - [ ] Accelerometer
 - [x] App Information
 - [ ] Battery
 - [ ] Clipboard
 - [ ] Compass
 - [ ] Connectivity
 - [ ] Data Transfer
 - [x] Device Information
 - [ ] File System Helpers
 - [ ] Email
 - [ ] Flashlight
 - [ ] Geocoding
 - [ ] Geolocation
 - [ ] Gyroscope
 - [ ] Magnetometer
 - [x] Open Browser
 - [ ] Phone Dialer
 - [ ] Preferences
 - [ ] Screen Lock
 - [ ] Secure Storage
 - [ ] SMS
 - [ ] Text-to-Speech
 - [x] Version Tracking
 - [x] Vibrate

### CODE REVIEW

**Comment by dend** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[Samples/Samples.Tizen/Program.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Samples/Samples.Tizen/Program.cs) | :white_check_mark:Succeeded | |
[Samples/Samples.Tizen/Samples.Tizen.csproj](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Samples/Samples.Tizen/Samples.Tizen.csproj) | :white_check_mark:Succeeded | |
[Samples/Samples.Tizen/shared/res/Samples.Tizen.png](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Samples/Samples.Tizen/shared/res/Samples.Tizen.png) | :white_check_mark:Succeeded | |
[Samples/Samples.Tizen/tizen-manifest.xml](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Samples/Samples.Tizen/tizen-manifest.xml) | :white_check_mark:Succeeded | |
[Xamarin.Essentials.sln](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials.sln) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Accelerometer/Accelerometer.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Accelerometer/Accelerometer.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/AppInfo/AppInfo.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/AppInfo/AppInfo.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Battery/Battery.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Battery/Battery.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Browser/Browser.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Browser/Browser.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Clipboard/Clipboard.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Clipboard/Clipboard.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Compass/Compass.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Compass/Compass.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Connectivity/Connectivity.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Connectivity/Connectivity.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/DataTransfer/DataTransfer.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/DataTransfer/DataTransfer.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/DeviceDisplay/DeviceDisplay.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/DeviceDisplay/DeviceDisplay.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/DeviceInfo/DeviceInfo.shared.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/DeviceInfo/DeviceInfo.shared.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/DeviceInfo/DeviceInfo.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/DeviceInfo/DeviceInfo.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Email/Email.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Email/Email.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/FileSystem/FileSystem.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/FileSystem/FileSystem.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Flashlight/Flashlight.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Flashlight/Flashlight.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Geocoding/Geocoding.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Geocoding/Geocoding.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Geolocation/Geolocation.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Geolocation/Geolocation.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Gyroscope/Gyroscope.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Gyroscope/Gyroscope.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Magnetometer/Magnetometer.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Magnetometer/Magnetometer.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Permissions/Permissions.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Permissions/Permissions.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/PhoneDialer/PhoneDialer.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/PhoneDialer/PhoneDialer.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Platform/Platform.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Platform/Platform.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Preferences/Preferences.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Preferences/Preferences.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/ScreenLock/ScreenLock.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/ScreenLock/ScreenLock.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/SecureStorage/SecureStorage.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/SecureStorage/SecureStorage.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Sms/Sms.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Sms/Sms.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/TextToSpeech/TextToSpeech.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/TextToSpeech/TextToSpeech.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Vibration/Vibration.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Vibration/Vibration.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Xamarin.Essentials.csproj](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Xamarin.Essentials.csproj) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C5%5C11%5Cf63aadd0-c8f5-13ca-04e7-037e45f8c0b9%5CPullRequest%5C201805111837313790-244%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=bH5C76igOTIfKs8bX07dQ%2B2YyCgVQS1FIJf%2FCYbHKQg%3D&st=2018-05-11T18%3A33%3A30Z&se=2018-06-11T18%3A38%3A30Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by dend** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[Samples/Samples.Tizen/Program.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Samples/Samples.Tizen/Program.cs) | :white_check_mark:Succeeded | |
[Samples/Samples.Tizen/Samples.Tizen.csproj](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Samples/Samples.Tizen/Samples.Tizen.csproj) | :white_check_mark:Succeeded | |
[Samples/Samples.Tizen/shared/res/Samples.Tizen.png](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Samples/Samples.Tizen/shared/res/Samples.Tizen.png) | :white_check_mark:Succeeded | |
[Samples/Samples.Tizen/tizen-manifest.xml](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Samples/Samples.Tizen/tizen-manifest.xml) | :white_check_mark:Succeeded | |
[Xamarin.Essentials.sln](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials.sln) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Accelerometer/Accelerometer.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Accelerometer/Accelerometer.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/AppInfo/AppInfo.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/AppInfo/AppInfo.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Battery/Battery.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Battery/Battery.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Browser/Browser.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Browser/Browser.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Clipboard/Clipboard.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Clipboard/Clipboard.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Compass/Compass.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Compass/Compass.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Connectivity/Connectivity.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Connectivity/Connectivity.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/DataTransfer/DataTransfer.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/DataTransfer/DataTransfer.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/DeviceDisplay/DeviceDisplay.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/DeviceDisplay/DeviceDisplay.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/DeviceInfo/DeviceInfo.shared.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/DeviceInfo/DeviceInfo.shared.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/DeviceInfo/DeviceInfo.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/DeviceInfo/DeviceInfo.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Email/Email.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Email/Email.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/FileSystem/FileSystem.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/FileSystem/FileSystem.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Flashlight/Flashlight.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Flashlight/Flashlight.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Geocoding/Geocoding.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Geocoding/Geocoding.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Geolocation/Geolocation.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Geolocation/Geolocation.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Gyroscope/Gyroscope.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Gyroscope/Gyroscope.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Magnetometer/Magnetometer.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Magnetometer/Magnetometer.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Permissions/Permissions.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Permissions/Permissions.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/PhoneDialer/PhoneDialer.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/PhoneDialer/PhoneDialer.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Platform/Platform.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Platform/Platform.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Preferences/Preferences.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Preferences/Preferences.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/ScreenLock/ScreenLock.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/ScreenLock/ScreenLock.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/SecureStorage/SecureStorage.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/SecureStorage/SecureStorage.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Sms/Sms.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Sms/Sms.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/TextToSpeech/TextToSpeech.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/TextToSpeech/TextToSpeech.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Vibration/Vibration.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Vibration/Vibration.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Xamarin.Essentials.csproj](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Xamarin.Essentials.csproj) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C5%5C11%5Cf63aadd0-c8f5-13ca-04e7-037e45f8c0b9%5CPullRequest%5C201805111840334262-244%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=oiEk2P2bOF4TChT1un8kGR%2F1BMU4n9WGpKfUp%2BhJetU%3D&st=2018-05-11T18%3A36%3A26Z&se=2018-06-11T18%3A41%3A26Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: Xamarin.Essentials/DeviceInfo/DeviceInfo.tizen.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    public static partial class DeviceInfo
+    {
+        static string GetModel()
+           => Plat.GetSystemInfo("model_name");
+
+        static string GetManufacturer()
+            => Plat.GetSystemInfo("manufacturer");
+
+        static string GetDeviceName()
+            => Plat.GetSystemInfo("device_name");
```
Comment (mattleibow) [inline Xamarin.Essentials/DeviceInfo/DeviceInfo.tizen.cs]: I noticed this did not return anything. Is this correct?

**Comment by dend** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[Samples/Samples.Tizen/Program.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Samples/Samples.Tizen/Program.cs) | :white_check_mark:Succeeded | |
[Samples/Samples.Tizen/Samples.Tizen.csproj](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Samples/Samples.Tizen/Samples.Tizen.csproj) | :white_check_mark:Succeeded | |
[Samples/Samples.Tizen/shared/res/Samples.Tizen.png](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Samples/Samples.Tizen/shared/res/Samples.Tizen.png) | :white_check_mark:Succeeded | |
[Samples/Samples.Tizen/tizen-manifest.xml](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Samples/Samples.Tizen/tizen-manifest.xml) | :white_check_mark:Succeeded | |
[Samples/Samples/View/BasePage.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Samples/Samples/View/BasePage.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials.sln](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials.sln) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Accelerometer/Accelerometer.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Accelerometer/Accelerometer.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/AppInfo/AppInfo.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/AppInfo/AppInfo.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Battery/Battery.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Battery/Battery.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Browser/Browser.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Browser/Browser.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Clipboard/Clipboard.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Clipboard/Clipboard.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Compass/Compass.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Compass/Compass.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Connectivity/Connectivity.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Connectivity/Connectivity.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/DataTransfer/DataTransfer.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/DataTransfer/DataTransfer.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/DeviceDisplay/DeviceDisplay.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/DeviceDisplay/DeviceDisplay.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/DeviceInfo/DeviceInfo.shared.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/DeviceInfo/DeviceInfo.shared.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/DeviceInfo/DeviceInfo.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/DeviceInfo/DeviceInfo.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Email/Email.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Email/Email.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/FileSystem/FileSystem.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/FileSystem/FileSystem.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Flashlight/Flashlight.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Flashlight/Flashlight.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Geocoding/Geocoding.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Geocoding/Geocoding.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Geolocation/Geolocation.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Geolocation/Geolocation.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Gyroscope/Gyroscope.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Gyroscope/Gyroscope.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Magnetometer/Magnetometer.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Magnetometer/Magnetometer.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Permissions/Permissions.shared.enums.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Permissions/Permissions.shared.enums.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Permissions/Permissions.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Permissions/Permissions.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/PhoneDialer/PhoneDialer.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/PhoneDialer/PhoneDialer.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Platform/Platform.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Platform/Platform.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Preferences/Preferences.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Preferences/Preferences.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/ScreenLock/ScreenLock.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/ScreenLock/ScreenLock.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/SecureStorage/SecureStorage.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/SecureStorage/SecureStorage.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Sms/Sms.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Sms/Sms.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/TextToSpeech/TextToSpeech.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/TextToSpeech/TextToSpeech.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Vibration/Vibration.tizen.cs](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Vibration/Vibration.tizen.cs) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Xamarin.Essentials.csproj](https://github.com/xamarin/Essentials/blob/feature/tizen-support/Xamarin.Essentials/Xamarin.Essentials.csproj) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C5%5C11%5Cf63aadd0-c8f5-13ca-04e7-037e45f8c0b9%5CPullRequest%5C201805112043166767-244%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=6eBCuI7qUhDoi8zHcIBzUaQTFBIEkZq3Z4Bn0R4aaZo%3D&st=2018-05-11T20%3A39%3A11Z&se=2018-06-11T20%3A44%3A11Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request closed without merging**